src/runtime/nuxt-image-mixins.ts

```diff
@@ -85,6 +95,18 @@ export default {
     }
   },
   computed: {
+    imageWidth () {
+      return this.width || this.imageW
+    },
+    imageHeight () {
+      return this.height || this.imageH
+    },
+    imageRatio () {
+      if (this.height && this.width) {
```

Comment:
Can we use new `imageWidth` computed to merge to conditions?

Reply:
It isn't safe to use `imageWidth` to calculate ratio, it leads to wrong ratio if user provides `width` prop and does not provide `height` prop

Reply:
But then Is it useful if user only sets one of `width` or `height`? (if not we may throw an error in created about wrong usage)

Reply:
I do refactor `imageRatio` property and manage to remove `imageWidth` & `imageHeight` computed because they have no real use in the component.
`width` and `height` are only used for calculating ratio and they are optional, missing one of them is like missing both of them. 
I think missing one of these props is not a critical error

Reply:
> width and height are only used for calculating ratio and they are optional, missing one of them is like missing both of them

I think we should at least warn users that prop is ignored instead of silently ignoring wrong configuration